src/main/java/edu/harvard/iq/dataverse/util/SignpostingResources.java

```diff
@@ -0,0 +1,337 @@
+    }
+
+    /**
+     * Get identifier schema for each author
+     * <p>
+     * For example:
+     * if author has VIAF
+     * Link: <http://viaf.org/viaf/:id/>; rel="author"
+     *
+     * @param datasetAuthors list of all DatasetAuthor object
+     * @return all the non empty author links in a string
+     */
+    private String getAuthors(List<DatasetAuthor> datasetAuthors) {
+        String singleAuthorString;
+        String identifierSchema = "";
+        int visibleAuthorCounter = 0;
+//        if (datasetAuthors.size() > maxAuthors) {return "";}
+        for (DatasetAuthor da : datasetAuthors) {
+            logger.info(String.format(
+                    "idtype: %s; idvalue: %s, affiliation: %s; identifierUrl: %s",
+                    da.getIdType(),
+                    da.getIdValue(),
+                    da.getAffiliation(),
+                    da.getIdentifierAsUrl()
+            ));
+
+            String authorURL = "";
+            authorURL = getAuthorUrl(da);
+            if (!Objects.equals(authorURL, "")) {
+                visibleAuthorCounter++;
+                // return empty if number of visible author more than max allowed
+                if (visibleAuthorCounter >= maxAuthors) return "";
+                singleAuthorString = "<" + authorURL + ">;rel=\"author\"";
+                if (Objects.equals(identifierSchema, "")) {
+                    identifierSchema = singleAuthorString;
+                } else {
+                    identifierSchema = String.join(",", identifierSchema, singleAuthorString);
+                }
+            }
+        }
+
+        logger.info(String.format("identifierSchema: %s", identifierSchema));
+        return identifierSchema;
+    }
+
+    /**
+     * Get key, values of signposting items and return as string
+     *
+     * @return comma delimited string
+     */
+    public String getLinks() {
+        List<String> valueList = new LinkedList<>();
+        Dataset ds = workingDatasetVersion.getDataset();
+
+        String identifierSchema = getAuthors(workingDatasetVersion.getDatasetAuthors());
+        if (!identifierSchema.equals("")) {
+            valueList.add(identifierSchema);
+        }
+
+        if (!Objects.equals(ds.getPersistentURL(), "")) {
+            String citeAs = "<" + ds.getPersistentURL() + ">;rel=\"cite-as\"";
+            valueList.add(citeAs);
+        }
+
+        List<FileMetadata> fms = workingDatasetVersion.getFileMetadatas();
+        String items = getItems(fms);
+        if (items != null && !Objects.equals(items, "")) {
+            valueList.add(items);
+        }
+
+        String describedby = "<" + describedByJsonObj.getString(ds.getProtocol()) + ds.getAuthority() + "/"
+                + ds.getIdentifier() + ">;rel=\"describedby\"" + ";type=\"" + describedByJsonObj.getString("type") + "\"";
+        describedby += ",<" + systemConfig.getDataverseSiteUrl() + "/api/datasets/export?exporter=schema.org&persistentId="
```

Comment:
Just curious - does Signposting prefer this export? Or would the more complete JSON or OAI_ORE exports be more relevant? (Or can the dataset be described by all the exports so you can find the format you understand?)

Reply:
This is the format described in the level 1 protocol. Might be a good idea check with Herbert.